### PR TITLE
Added API endpoint: GET /api/v1/server_info

### PR DESCRIPTION
**Pull request by ajiprabowo**

### CODE REVIEW

File: pkg/server/server.go [hunk 1]
```diff
@@ -127,6 +127,8 @@ func (s *Server) setupRouter() (*httprouter.Router, error) {
 
 	router.GET("/healthz", s.handleHealthz)
 
+	router.GET("/api/v1/server_info", s.getServerInfo)
```
Comment (andylibrian) [inline pkg/server/server.go:130]: For the sake of consistency

```suggestion
	router.GET("/api/v1/server_info", s.handleServerInfo)
```

File: pkg/server/server.go [hunk 2]
```diff
@@ -341,3 +343,16 @@ func (s *Server) handleStopLoadTest(responseWriter http.ResponseWriter, req *htt
 func (s *Server) handleHealthz(responseWriter http.ResponseWriter, req *http.Request, _ httprouter.Params) {
 	responseWriter.WriteHeader(200)
 }
+
+func (s *Server) getServerInfo(responseWriter http.ResponseWriter, req *http.Request, _ httprouter.Params) {
```
Comment (andylibrian) [inline pkg/server/server.go:347]:
```suggestion
func (s *Server) handleServerInfo(responseWriter http.ResponseWriter, req *http.Request, _ httprouter.Params) {
```

File: pkg/server/server.go [hunk 3]
```diff
@@ -341,3 +343,16 @@ func (s *Server) handleStopLoadTest(responseWriter http.ResponseWriter, req *htt
...
+	// Server Info (Body: Json Content-type)
```
Comment (andylibrian) [inline pkg/server/server.go:348]: The code is clear enough that this comment is unnecessary
```suggestion
```

File: pkg/server/server.go [hunk 4]
```diff
@@ -341,3 +343,16 @@ func (s *Server) handleStopLoadTest(responseWriter http.ResponseWriter, req *htt
...
+	serverInfo := messages.ServerInfo{NumOfWorkers: len(s.workerService.workers), State: loadTestStateToString(s.loadTestState)}
+	serverInfoMsg, _ := json.Marshal(serverInfo)
+
+	//Header: Json Content-type
```
Comment (andylibrian) [inline pkg/server/server.go:352]: The code is clear enough that this comment is too verbose
```suggestion
```

**Review by andylibrian — CHANGES_REQUESTED** [review]

File: pkg/server/server.go (hunk 1 above)
Comment (ajiprabowo) [reply pkg/server/server.go:130]: Noted

**Review by andylibrian — APPROVED** [review]

**Pull request merged**